### ISSUE TITLE
filename-format break on single taged frame within file

### ISSUE THREAD

**Issue by bfloch** [aseprite/aseprite#1107]
Exporting a file with a single frame (with tag) does break the filename-format.
```
aseprite --batch title_v005.ase --filename-format "{tag}_{tagframe0000}.{extension}" --save-as output.png
```
With two frames, tagged as `default` I get
```
default_0001.png
default_0001.png
```

With one frame I get
```
_.png
```

**Comment by bfloch** [comment]
Probably related: Neither --frame-tag nor --frame-range seem to work when there is only one frame per tag.
```
aseprite --batch sogfont_v002.ase --frame-tag "P" --frame-range 41,41 --filename-format P_0000.{extension} --layer _CROPBOX --save-as output.png
```

This will still export the hole frame range. So my workaround of reading the data and exporting per tag does not work in this case :-/

**Comment by bfloch** [comment]
Best workaround I found is to add an empty frame for each one frame tag and delete it afterwards.

**Comment by dacap** [comment]
This will be fixed in v1.2-beta1
